models/PyCryptoBot.py

```diff
@@ -366,16 +366,16 @@ def getTime(self):
             return ''
 
     def isLive(self):
-        return self.is_live
+        return self.is_live == 1
 
     def isVerbose(self):
-        return self.is_verbose
+        return self.is_verbose == 1
 
     def shouldSaveGraphs(self):
-        return self.save_graphs
+        return self.save_graphs == 1
 
     def isSimulation(self):
-        return self.is_sim
+        return self.is_sim == 1
 
     def isTelegramEnabled(self):
```

Comment:
Should this one be changed too?

Reply:
Yep, `is_sim` is an int so `isSimulation` should be a boolean.

Reply:
No, I meant the `isTelegramEnabled()`

Reply:
Ha sorry.
No because `self.telegram` is already a boolean so `isTelegramEnabled()` too